examples/src/main/java/io/zenoh/ZSubThr.java

```diff
@@ -59,19 +57,18 @@ public static void report() {
                 ": averaged " + avg + " msgs/sec");
     }
 
-    public static void main(String[] args) throws ZenohException {
+    public static void main(String[] args) throws ZenohException, InterruptedException {
         System.out.println("Opening Session");
         try (Session session = Session.open()) {
             try (KeyExpr keyExpr = KeyExpr.tryFrom("test/thr")) {
                 try (Subscriber<Unit> subscriber = session.declareSubscriber(keyExpr).with(sample -> listener()).res()) {
-                    Scanner scanner = new Scanner(System.in);
-                    while (!scanner.nextLine().equals("q")) {
-                        // Do nothing
+                    System.out.println("Press CTRL-C to quit...");
+                    while (true) {
+                        Thread.sleep(1000);
                     }
-                    scanner.close();
                 }
             }
         }
-        report();
+        // report();
```

Comment:
Why commenting this call?

Reply:
Report function is unreachable, and doesn't exist in ZSubThr examples on other bindings.
Removing it.

Reply:
`report` function exists in other bindings but is no longer called after example alignement.
In the future, examples should be updated to call their respective `report` function.
Marking function with `TODO`.